subprojects/testfx-core/src/main/java/org/testfx/robot/DragRobot.java

```diff
@@ -22,15 +22,40 @@
 
 public interface DragRobot {
 
+    /**
+     * Presses the given mouse button(s) on whatever is under the mouse's current location.
+     *
+     * @param buttons the mouse buttons to press
+     */
     public void drag(MouseButton... buttons);
 
+    /**
+     * Moves the mouse to {@link PointQuery#query()} before presses the given mouse button(s) on whatever is under
+     * the mouse's new location.
+     *
+     * @param pointQuery the pointQuery
+     * @param buttons the mouse buttons to press
+     */
     public void drag(PointQuery pointQuery,
                      MouseButton... buttons);
 
+    /**
+     * Releases the mouse at current position.
+     */
     public void drop();
 
+    /**
+     * Moves the mouse to {@link PointQuery#query()} before releasing the mouse.
+     *
+     * @param pointQuery the pointQuery
+     */
     public void dropTo(PointQuery pointQuery);
 
+    /**
+     * Moves the mouse horizontally by {@code x} and vertically by {@code y} before releasing the mouse.
+     * @param x the amount by which to move the mouse horizontally
```

Comment:
Missing space between description and `@param` tags (guess our extensive checkstyle rules doesn't check that, heh).